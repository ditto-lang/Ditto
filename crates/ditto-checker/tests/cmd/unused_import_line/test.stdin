module Test exports (..);

import Dep;

five = 5;
