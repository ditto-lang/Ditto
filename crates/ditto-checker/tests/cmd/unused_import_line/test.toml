bin.name = "ditto-checker-testbin-check-module"
args = ["unused_import_line.ditto"]
fs.sandbox = true
