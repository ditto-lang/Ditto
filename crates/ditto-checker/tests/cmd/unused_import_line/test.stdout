{
  "module_name": [
    "Test"
  ],
  "exports": {
    "types": {},
    "constructors": {},
    "values": {
      "five": {
        "doc_comments": [],
        "doc_position": 0,
        "value_type": {
          "type": "PrimConstructor",
          "data": "Int"
        }
      }
    }
  },
  "types": {},
  "constructors": {},
  "values": {
    "five": {
      "doc_comments": [],
      "name_span": {
        "start_offset": 38,
        "end_offset": 42
      },
      "expression": {
        "expression": "Int",
        "data": {
          "span": {
            "start_offset": 45,
            "end_offset": 46
          },
          "value": "5",
          "value_type": {
            "type": "PrimConstructor",
            "data": "Int"
          }
        }
      }
    }
  },
  "values_toposort": [
    "five"
  ]
}